core/src/test/java/org/elasticsearch/common/io/StreamsTests.java

```diff
@@ -81,7 +81,7 @@ public void testCopyToString() throws IOException {
         String result = copyToString(in);
         assertThat(result, equalTo(content));
     }
-    
+
```

Comment:
These are really hard to remove, my intellij did that when auto import new libs. But I did that ;)